examples/dart/test/read_write_data_test.dart

```diff
@@ -95,6 +95,18 @@ void main() {
       // :snippet-end:
       expect(team.name, 'Millennium Falcon Crew');
       expect(humanCrewMembers.length, 3);
+
+      // :snippet-start: filter-iterable
+      final listOfNames = ['Luke', 'Leia'];
+      final matchingRealmObjects =
+          realm.query<Person>(r'name IN $0', [listOfNames]);
```

Comment:
I am assuming this is a typo? (Although if it is, I'm a little surprised we don't get an error.)
```suggestion
          realm.query<Person>('name IN $0', [listOfNames]);
```

Reply:
I actually am not sure exactly what it does. I should ask the Flutter team. They use it in their tests instead of escaping `$` for paramerterized queries.

However, since we use the `\$0` construction in the rest of the Flutter docs, I'm going to change to that.